src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePump.cs

```diff
@@ -408,7 +408,7 @@ public override async Task StopAsync(CancellationToken cancellationToken)
         private async Task DeleteTopicSubscriptionAsync(CancellationToken cancellationToken)
         {
             ServiceBusAdministrationClient serviceBusClient = await Settings.GetServiceBusAdminClientAsync();
-            string entityPath = await Settings.GetEntityPathAsync();
+            string entityPath = Settings.EntityName;
```

Comment:
In the past, also this call would run the authentication, now that's also different and populated implicitly.